src/shared/canvas/MyData/types.ts

```diff
@@ -45,3 +45,26 @@ export type TDate = {
 };
 export type TDateOptions = 'singleDate' | 'dateStart' | 'dateEnd';
 export const DATE_PATTERN = 'DD/MM/YYYY';
+export type TType = {
+  key: string;
+  value: string;
+  label: string;
+  docCount: number;
+};
+
+export type TTypeAggregationsResult = {
+  '@context': string;
+  total: number;
+  aggregations: {
+    projects: TTypesAggregatedProperty;
+    types: TTypesAggregatedProperty;
+  };
+};
+
+export type TTypesAggregatedProperty = {
```

Comment:
Nice to have: There are `AggregatedProperty` and `AggregationsResult` types defined in `DataExplorerUtils.tsx`  file. They look very similar to the `TTypesAggregatedProperty` and `TTypeAggregationsResult`. We can remove one of them and use the same types in both places.